### ISSUE TITLE
Duplicate Align datum while creating sig-pickle

### ISSUE THREAD

**Issue by hvgazula** [hassonlab/247-pickling#107]
Avoid duplication 
![image](https://user-images.githubusercontent.com/9400767/207090867-8e647e10-8026-44de-a00d-809b80283c0e.png)

**Comment by hvgazula** [comment]
Alignment happens every time a new subject is encountered (podcast). On that note, do we ever have to create significant electrode pickles for 247? @VeritasJoker

**Comment by VeritasJoker** [comment]
We never used significant electrode pickle fo 247

**Comment by hvgazula** [comment]
I know we did not do it in the past, but the question is about the future. We will have to?

**Comment by VeritasJoker** [comment]
I don't think so. We always run all electrodes for 247 subjects because we never knew which eletrodes we will end up using

**Comment by hvgazula** [comment]
Okay, let's keep this open for a bit. I believe Tom is working on selecting significant electrodes for 247.

**Comment by zkokaja** [comment]
24/7 shouldn't be a problem since each subject has its own datum
